views/room/calendar.pug

```diff
@@ -1,23 +1,18 @@
 extend ../layouts/main
 
 block styles
-  link(rel="stylesheet", href="https://cdn.jsdelivr.net/npm/@fullcalendar/core@4.3.1/main.min.css")
-  link(rel="stylesheet", href="https://cdn.jsdelivr.net/npm/@fullcalendar/daygrid@4.3.0/main.min.css")
-  link(rel="stylesheet", href="https://cdn.jsdelivr.net/npm/@fullcalendar/timegrid@4.3.0/main.min.css")
+  link(rel="stylesheet", href="https://cdn.jsdelivr.net/npm/fullcalendar@5.6.0/main.min.css")
 
 block content
   div(class="ml-4")
     select(class='rounded form-element' onchange='window.location.href=this.value')
       each floor in ROOMS
         option(value=`${floor}` selected=floor==room ? 'selected' : false)
           a(href=`/rooms/${floor}`)= floor + ". emelet"
-  div(class='flex flex-col items-center justify-center')
-    div(class='max-w-6xl p-3 mx-4 my-4 sm:p-6 rounded-2xl bg-default shadow-full')
+  div(class='max-w-6xl p-3 mx-auto my-4 sm:p-6 rounded-2xl bg-default shadow-full')
```

Comment:
On mobile there should be a small margin on the right / left of this div so it won't touch the edge of the screen. But we won't need it on larger screens, so we can use an approach like `mx-4 sm:mx-auto`.